### DISCUSSION TITLE
How to run existing local projects without docker?

### DISCUSSION THREAD

**Discussion by Mage2developer** [markshust/docker-magento#744] [Q&A]
Hi @markshust,

I have followed the steps to install and set up a new project from **Automated Setup (New Project)**, and the project setup was done successfully with sample data. After that to run my local projects I tried to open my local URL(http://local.ce243.com/), the url not working and gives the following error:
![not-working-project](https://user-images.githubusercontent.com/26230770/180946464-6adb1965-f6df-4193-877b-8b3b1f51f24f.png)

So I start apache2, mysql, elasticsearch, and redis-server services using the following command, but no luck. It shows the same above error:

`sudo service apache2 start && sudo service mysql start && sudo service elasticsearch start && sudo service redis-server start`

Please let me know how to overcome this issue. Thanks in anticipation.

**Comment by markshust** [comment]
Unfortunately I'd need more info in order to get back to you about this issue. However, you'll never want to run local services along with this project -- it's one or the other.

I'd recommend checking out the free course I made about this project at https://m.academy/courses/set-up-magento-2-development-environment-docker/ which will guide you through every step of the process. If something doesn't jive along the way, you'll need to take note of any errors happening in the terminal, which will provide you some info to work against.
